src/material-experimental/mdc-table/_table-theme.scss

```diff
@@ -1,7 +1,7 @@
 @use 'sass:map';
 @use '@material/theme/theme-color' as mdc-theme-color;
 @use '@material/data-table/data-table' as mdc-data-table;
-@use '@material/data-table/data-table-theme' as mdc-data-table-theme;
+@use '@material/data-table' as mdc-data-table-theme;
```

Comment:
Can this be combined with the `data-table/data-table` import above? The same question applies to the tooltip change.

Reply:
Oddly enough, it can't. I think because they do not intend to export core styles to clients, just theme mixins. So we have to import themes from their "_index.scss" and core styles directly from their definitions

Data table: https://github.com/material-components/material-components-web/blob/master/packages/mdc-data-table/_index.scss
Tooltip: https://github.com/material-components/material-components-web/blob/master/packages/mdc-tooltip/_index.scss

Reply:
That seems a little weird that they allow one deep import, but not the other. It's not a blocker for this PR though.